### DISCUSSION TITLE
Can't resolve '../internals/*'

### DISCUSSION THREAD

**Discussion by rexkenley** [babel/babel#15021] [General] [answered]
I am getting a strange error when generating my code artifacts using webpack. But when I went to look for the "missing" modules, it is there.

Example

ERROR in ../../../node_modules/@babel/runtime-corejs3/regenerator/index.js 3:14-54
Module not found: Error: Can't resolve '../helpers/regeneratorRuntime' in 'C:\Temp10\Ahab\node_modules\@babel\runtime-corejs3\regenerator'    
 @ ./src/lib/dependent-option-sets.ts 2:0-69 105:42-66 108:11-35

![image](https://user-images.githubusercontent.com/10548251/194168871-4db4d593-3fef-42ab-9f59-f8833aead456.png)

![image](https://user-images.githubusercontent.com/10548251/194169096-1a79c107-bcb8-4966-9842-667d049ae729.png)

**Comment by nicolo-ribaudo** [comment, marked as answer]
If the files are there, this is probably a webpack bug 🤔

**Reply by rexkenley** [reply]
I was able to resolve the issue by adding js to webpack config resolve extensions. 

Thanks @nicolo-ribaudo
